### PR TITLE
Feature/logback

### PR DESCRIPTION
**Pull request by czyzu0507**
as in git message only logback configuration

### CODE REVIEW

File: robot-service/src/main/resources/logback.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <file>logback.log</file>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{5} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="DEBUG">
+        <appender-ref ref="STDOUT"/>
+    </root>
+</configuration>
```
Comment (danone771909) [inline robot-service/src/main/resources/logback.xml:19]: :+1:

File: pom.xml [hunk 2]
```diff
@@ -33,6 +33,9 @@
         <json-simple.version>1.1.1</json-simple.version>
         <common.dbcp.version>1.4</common.dbcp.version>
         <hibernate.version>5.2.2.Final</hibernate.version>
+        <lombock.version>1.16.10</lombock.version>
```
Comment (LIttleAncientForestKami) [inline pom.xml]: Lombock is different from Lombok how? :smiling_imp:

File: pom.xml [hunk 3]
```diff
@@ -119,8 +122,38 @@
             <artifactId>spring-orm</artifactId>
             <version>${springframework.version}</version>
         </dependency>
+
+        <!--lombock-->
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombock.version}</version>
```
Comment (LIttleAncientForestKami) [inline pom.xml]: Kinda shame, that we have a lombock, but we don't have a logbak. :smiling_imp:

**Comment by LIttleAncientForestKami** [comment]
It's almost :ok:, just one typo.

**Comment by danone771909** [comment]
It's done :+1:

**Pull request merged**